src/Orchard.Web/Modules/Orchard.Azure.MediaServices/Views/Parts/CloudVideo.cshtml

```diff
@@ -1,9 +1,21 @@
 ﻿@using Orchard.ContentManagement;
 
+@{
+    string alternateText = string.Empty;
+    try
```

Comment:
This is not a nullability issue. The original code was failing when embedding a cloud media video item in a page because of a binder issue with the As<> extension method.

This is a quick workaround for the existing error in the code. I think that doing a IContent cast before calling the As<> extension method will fix it; I will push the changes later.